ci/azure-pipelines-k8s.yml

```diff
@@ -15,17 +15,13 @@ schedules:
     always: true
 
 variables:
-  - group: artifactory
-  - name: FABRIC_CFG_PATH
-    value: $(Agent.BuildDirectory)/go/src/github.com/hyperledger/fabric-test/config
-  - name: GOPATH
-    value: $(Agent.BuildDirectory)/go
-  - name: PATH
-    value: $(Agent.BuildDirectory)/go/bin:$(Agent.BuildDirectory)/go/src/github.com/hyperledger/fabric-test/bin:/bin:/usr/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin
-  - name: GO_VER
-    value: 1.13.8
-  - name: NODE_VER
-    value: 10.x
+  FABRIC_CFG_PATH: $(Agent.BuildDirectory)/go/src/github.com/hyperledger/fabric-test/config/
+  GO_BIN: $(Agent.BuildDirectory)/go/bin
+  GO_PATH: $(Agent.BuildDirectory)/go
+  GO_VER: 1.14.2
+  NODE_VER: 12.15.0
+  PATH: $(Agent.BuildDirectory)/go/bin:$(Agent.BuildDirectory)/go/src/github.com/hyperledger/fabric-test/bin:/bin:/usr/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin
+  group: artifactory
```

Comment:
This won't work. When you use a variable group `group: factory` the other entries need to be in the form of:
```
- name: GOPATH	  GO_VER: 1.14.2
   value: $(Agent.BuildDirectory)/go
```

But let me take a look at the failure and see what I can figure out